pkg/ocm/client/client.go

```diff
@@ -194,6 +196,7 @@ func (c *OCMClient) NewShare(ctx context.Context, endpoint string, r *NewShareRe
 	if err != nil {
 		return nil, err
 	}
+	log.Debug().Msgf("OCM send to %s: %s", url, body)
```

Comment:
Can you add the url and body as key-values in the logs, instead of including directly in the message?
Something like `log.Debug().Str("url", url).Msg("<something>")`